proposed/http-client/http-client.md

```diff
@@ -43,10 +43,9 @@ interface HttpClient
     /**
      * Sends a PSR-7 request.
      *
-     * If a request is sent without any prior configuration, an exception MUST NOT be thrown
-     * when a response is recieved, no matter the HTTP status code.
+     * An exception MUST NOT be thrown when a response is recieved, no matter the HTTP status code.
      *
-     * If a request is sent without any prior configuration, a HTTP client MUST NOT follow redirects.
+     * The HTTP client MUST NOT follow redirects.
```

Comment:
I guess you probably don't want an exception, but instead log the redirect, so it can be reviewed?

With an exception you instantly break the app that is supposed to continue working with the redirect.